lxd/util/fs_64bit.go

```diff
@@ -1,4 +1,4 @@
-// +build amd64 ppc64 ppc64le arm64 s390x
+// +build amd64 ppc64 ppc64le arm64 s390x mips64 mips64le
```

Comment:
Maybe we could document arches that we as upstream support and others which are supported by the community? @stgraber wdyt?

Reply:
That's kinda up to the distros as far as what they want to support. As upstream LXD, I'm fine accepting fixes for various distros.

As the packager for the snap, I know what architecture I build and test for.

It's a bit odd having the two hats but I don't think we need a stronger stance around that.
Right now as upstream, our CI is pretty much x86 only, for non-x86, it's effectively snap CI. Just happens to be run by the same people ;)